hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/FacilityInitModule.java

```diff
@@ -123,6 +124,9 @@ static FacilityInitializer initFacilities(
                 final var bootstrapConfig = bootstrapConfigProvider.getConfiguration();
                 exchangeRateManager.init(state, schema.genesisExchangeRates(bootstrapConfig));
                 feeManager.update(schema.genesisFeeSchedules(bootstrapConfig));
+                //                if (bootstrapConfig.getConfigData(FeesConfig.class).simpleFeesEnabled()) {
```

Comment:
We should keep this behind feature flag. Probably better to check feature flag inside `updateSimpleFees` because its used in multiple places

Reply:
Hmmm I don't understand. Why can't we enable the flag just for that one test that is failing?